RSFormView/Classes/Extensions/StringExtension.swift

```diff
@@ -14,8 +14,10 @@ extension String {
     switch type {
     case .email:
       isValid = isEmailFormatted()
-    case .numeric:
+    case .integer:
       isValid = isInteger()
+    case .double:
```

Comment:
We could make this enum take the max number of decimal places as parameter that way we don’t need to hardcore the 2 below

Reply:
> I really like the addition and the approach taken, would you like to make the changes to make the max decimals places configurable? Otherwise we can do the improvement later. Thanks so much for contributing @chumps52! I hope you’re finding RSFormView useful.

Thanks! It is very useful :)

I tried making the changes you suggested but not 100% sure - I could give it another go tomorrow but it might be best leaving it to someone who knows what they're doing if that's ok!

Reply:
I’m gonna merge your PR tomorrow and tag you on the PR addressing this so you can see the implementation. Thanks again for contributing!

Reply:
Great, thank you! And no problem :)